build/pipelines/templates/extensions-ci.yml

```diff
@@ -8,7 +8,7 @@ parameters:
   default: false
 - name: Solution
   type: string
-  default: not-specified
+  default:
```

Comment:
Is the explicit default here required?

Reply:
yes, if I remove `default:` completely then getting this error when trying to run the pipeline - _A value for the 'Solution' parameter must be provided._